package.json

```diff
@@ -30,14 +30,14 @@
     "deploy": "yarn run build && gh-pages --add --dist build/",
     "deploy-dev":
       "PUBLIC_URL=$npm_package_homepage_dev yarn run deploy --dest dev/",
-    "lint": "prettier --list-different src/**/*.js",
-    "lint-fix": "yarn lint --write",
+    "lint": "prettier --list-different src/**/*.js | scripts/lint_problems.sh",
+    "lint-fix": "prettier --write src/**/*.js",
```

Comment:
would you like to add `src/**/*.css` to those two scripts?

Reply:
Should we use `src/**/*.*s`?

Reply:
See updated push to this PR.

Reply:
`src/**/*.*s` sounds far too likely to have false postives. In my `~/src` directory, I have 56 types of files that would match that pattern that aren't css or js. It saves a couple of bytes, but it makes it much less clear what you're actually trying to do.

```
fd -t f 's$' | grep -v '.js$' | grep -v '.css$' | grep -o '\.[^.]*$' | sort | uniq | wc -l
```

Reply:
@mythmon Yeah I agree. I changed it to specify `src/**/*.js src/**/*.css` but then I discovered that Prettier can do .md as well. And it can do .json too. Arguably I'm not sure they're nearly as important to be part of the CI.